core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -851,6 +851,14 @@ public enum Property {
           + " the tablet server this duration to revive before reassigning its tablets"
           + " to other tablet servers."),
 
+  TABLE_LOCATION_MODE("table.location.mode", "locality", PropertyType.LAST_LOCATION_MODE,
+      "Describes how the system will assign tablets initially."
+          + " If 'locality' is the mode, then the system will assign tablets based on the data locality (e.g. the last major compaction location)."
+          + " If 'assignment' is the mode, then tablets will be initially assigned to the last place they were assigned which could be"
+          + " different then where they were last compacted given balancing activities."
+          + " Also note that master.startup.tserver properties might need to be set as well to ensure"
```

Comment:
Note on merging this forward into 2.1: the term `master` has changed to `manager` in newer versions, and this should be updated when this is merged forward.

```suggestion
          + " Also note that master.startup.tserver properties might need to be set as well to ensure"
```

Reply:
of course